kyo-core/shared/src/main/scala/kyo/chunks.scala

```diff
@@ -256,7 +258,7 @@ sealed abstract class Chunk[T] derives CanEqual:
                 else
                     acc
 
-            val unnested = new Array[U](totalSize())(using ClassTag(classOf[Any]))
+            val unnested = new Array[U](totalSize())(using ClassTag.Any.asInstanceOf[ClassTag[U]])
```

Comment:
Is there a reason the given doesn't work here? Perhaps it could be named?

Reply:
Do you mean the local implicit? It's for `A` but we need `U` here. By the way let me fix the generic name 😅

Reply:
ah, the prelude version already has it as `B`